lib/ui.rb

```diff
@@ -0,0 +1,28 @@
+module UI
+
+puts "".center(40, "X")
+puts "   WELCOME TO TIC TAC TOE!   ".center(40, "X")
+puts "".center(40, "X")
+
+
+WIN_COMBINATIONS = [[0,1,2], [3,4,5], [6,7,8], 
```

Comment:
Is there no way of generating this without hard coding it? I think there is!
We shall refactor this at later date